ps_spec.yml

```diff
@@ -29,17 +29,18 @@ stages:
               
               PKGNAME=$(node -e 'console.log(require("./package.json").name)')
               PKGVERSION=$(node -e 'console.log(require("./package.json").version)')
-              V_VERSION=v$PKGVERSION
               WORDCOUNT=$(npm view "${PKGNAME}@${PKGVERSION}" | wc -c)
               
-              git tag -a $V_VERSION -m "$PKGNAME release $V_VERSION" HEAD
-              
+              git config --global credential.helper \'store\' || true
+              echo "https://$ALPS_GH_USER:$ALPS_GH_TOKEN@github.com" > ~/.git-credentials
+              git tag -a $PKGVERSION -m "$PKGNAME release $PKGVERSION" HEAD
+ 
               npm install
               
               if [ "$WORDCOUNT" -eq 0 ]; then
                  echo "PUBLISHING $PKGNAME $PKGVERSION"
                  npm run lint
-                 git push origin $V_VERSION
+                 git push origin $PKGVERSION
```

Comment:
Dont oyu actually have to make the tag before you push?

`git tag -a -m 'Some message' '$PKGVERSION'`

Reply:
it is done at line 36